compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/repositories/RemotePackageRepository.java

```diff
@@ -99,8 +101,14 @@ public static RemotePackageRepository from(Environment environment, Path cacheDi
         }
 
         List<PackageVersion> packageVersions = new ArrayList<>();
-        for (String version : this.client.getPackageVersions(orgName, packageName)) {
-            packageVersions.add(PackageVersion.from(version));
+        try {
+            for (String version : this.client.getPackageVersions(orgName, packageName)) {
+                packageVersions.add(PackageVersion.from(version));
+            }
+        } catch (CentralClientException e) {
+            throw new ProjectException(e.getMessage());
+        } catch (ConnectionErrorException e) {
+            // ignore connect to remote repo failure
```

Comment:
are we continuing offline here? If so shall we log a message to say that?

Reply:
@hevayo @sameerajayasoma We decided not to log a warning message here right? Pls confirm.

Reply:
Let's see if how this works. we can add the log as an improvement later.

Reply:
Yes lets add as an improvement